lib/common/common.go

```diff
@@ -107,9 +107,9 @@ func GenerateManifest(layerData types.DockerImageData, dockerURL *types.ParsedDo
 
 	appURL := ""
 	// omit docker hub index URL in app name
-	if dockerURL.IndexURL != defaultIndex {
-		appURL = dockerURL.IndexURL + "/"
-	}
+	//if dockerURL.IndexURL != defaultIndex {
```

Comment:
If we need to remove the defaultIndex because we have several kinds now, we can remove this instead of commenting it out.